README.md

```diff
@@ -1,11 +1,15 @@
-# Chorizo (C#/.NET) - v0.1  
+# Chorizo (C#/.NET) - v0.5  
```

Comment:
This is the version of the software? Is there a changelog ?

Reply:
That is the intention, there is not a changelog at this time.